samples/web/content/apprtc/README.md

```diff
@@ -0,0 +1,73 @@
+# AppRTC Demo Code
+
+## Development
+
+Detailed information on devloping in the [GoogleChrome/webrtc](https://github.com/GoogleChrome/webrtc) github repo can be found in the [WebRTC GitHub repo developer's guide](https://docs.google.com/document/d/1tn1t6LW2ffzGuYTK3366w1fhTkkzsSvHsBnOHoDfRzY/edit?pli=1#heading=h.e3366rrgmkdk).
+
+The development AppRTC server can be accessed by visiting [http://localhost:8080](http://localhost:8080).
+
+Running AppRTC locally requires the [Google App Engine SDK for Python](https://cloud.google.com/appengine/downloads#Google_App_Engine_SDK_for_Python) and [Grunt](http://gruntjs.com/).
+
+Detailed instructions for running on Ubuntu Linux are provided below.
+
+### Running on Ubuntu Linux
+
+Install grunt by first installing [npm](https://www.npmjs.com/),
```

Comment:
I think all the grunt installing related stuff can be replaced by a link to http://gruntjs.com/getting-started.